python/pyspark/sql/functions.py

```diff
@@ -798,7 +798,7 @@ def factorial(col):
 # ---------------  Window functions ------------------------
 
 @since(1.4)
-def lag(col, count=1, default=None):
+def lag(col, offset=1, default=None):
```

Comment:
You need to update the param doc below too, but yes this seems fine.
The issue is that this potentially breaks a caller that specifies the arg by name, but we can do that in Spark 3.

Reply:
Oh, please file a JIRA for this one, actually, as we do need to add this to the release notes as a very minor breaking change.

Reply:
@srowen Done!